modules/administration-guide/pages/collecting-logs-using-chectl.adoc

```diff
@@ -5,11 +5,38 @@
 [id="collecting-logs-using-{prod-cli}_{context}"]
 = Collecting logs using {prod-cli}
 
+An installation of {prod} consists of several containers running in the {orch-name} cluster. While it is possible to manually collect logs from each running container, `{prod-cli}` provides commands which automate the process.
+
 Following commands are available to collect {prod} logs from the {orch-name} cluster using the `{prod-cli}` tool:
 
+`{prod-cli} server:logs`:: 
+Collects existing {prod} server logs and stores them in a directory on the local machine. By default, logs are downloaded to a temporary directory on the machine. However, this can be overwritten by specifying the `-d` parameter. For example, to download Che logs to the `/home/user/che-logs/` directory, use the command
++
+[source,shell,subs="+attributes"]
+----
+{prod-cli} server:logs -d /home/user/che-logs/
+----
++
+When run, `{prod-cli} server:logs` prints a message in the console specifying the directory that will store log files:
++
+[subs="+attributes"]
+----
+{prod} logs will be available in '/tmp/chectl-logs/1648575098344'
+----
++
+If {prod} is installed in a non-default {orch-namespace}, `{prod-cli} server:logs` requires the `-n <NAMESPACE>` paremeter, where `<NAMESPACE>` is the {platforms-namespace} in which {prod} was installed. For example, to get logs from {prod-short} in the `my-namespace` {orch-namespace}, use the command
++
+[source,shell,subs="+attributes"]
+----
+{prod-cli} server:logs -n my-namespace
+----
+
 `{prod-cli} server:deploy`:: 
-Automatically starts collecting logs during {prod-short} installation.
+Logs are automatically collected during {prod-short} installation when installed using `{prod-cli}`. As with `{prod-cli} server:logs`, the directory logs are stored in can be specified using the `-d` parameter.
```

Comment:
```suggestion
Using `{prod-cli}`, logs are automatically collected during {prod-short} installation. However, for `{prod-cli} server:logs`, the directory logs that are stored can be specified using the `-d` parameter.
```

Reply:
I think I prefer the original wording
> Logs are automatically collected during {prod-short} installation when installed using `{prod-cli}`

centers the explanation on the installation of Che rather than using the tool, but the article is about collecting logs via the chectl tool. Also, replacing "As with" with "However, for" inverts the meaning of the sentence -- the purpose of this paragraph is to mention that by default, logs are collected in the same way during `chectl server:deploy`, and the same configuration options apply (using `-d` to specify the logs directory).